jac-uk/admin
# 1307 activity logging

**Author checklist**


---
## What's included?
- Log an event when an exercise is published or unpublished

## Who should test?
✅ Product owner
✅ Developers


## How to test?
- Log into the admin portal and make changes to an exercise to trigger an event being logged
- Go to the Events page (via the menu item in the top right of the page) and check that an event has been logged.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Include screen grabs, notes etc.

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
